app/controllers/sessions_controller.rb

```diff
@@ -14,5 +14,10 @@ def create
     end
     redirect_to root_path
   end
-  
+
+  def destroy
+    session[:user_id] = nil
```

Comment:
Okay… so I [couldn't find documentation](http://apidock.com/rails/ActionController/RackDelegation/reset_session) on this, but I _swear_ it exists: `reset_session` dumps out the entire session which I prefer since there might be stuff you stored in `session` that you didn't remember. Plus `nil`ing out everything is a pain.